pkg/cmd/roachtest/test_runner.go

```diff
@@ -1109,14 +1109,10 @@ func (r *testRunner) serveHTTP(wr http.ResponseWriter, req *http.Request) {
 		var clusterName, clusterAdminUIAddr string
 		if w.Cluster() != nil {
 			clusterName = w.Cluster().name
-			adminUIAddrs, err := w.Cluster().ExternalAdminUIAddr(
-				req.Context(),
-				w.Cluster().Node(1),
-			)
-			if err != nil {
-				w.Cluster().t.Fatal(err)
+			adminUIAddrs, err := w.Cluster().ExternalAdminUIAddr(req.Context(), w.Cluster().Node(1))
+			if err == nil {
+				clusterAdminUIAddr = adminUIAddrs[0]
```

Comment:
Should we set clusterAdminUIAddr to "AdminUI URL not found" or something by default so the test result string is more clear in this case below if we can't grab a URL here?

Otherwise LGTM.